roles/bootstrap-os/tasks/bootstrap-archlinux.yml

```diff
@@ -0,0 +1,32 @@
+---
+#  raw: cat /etc/issue.net | grep '{{ bootstrap_versions }}'
+
+- name: Bootstrap | Check if bootstrap is needed
+  raw: which "{{ item }}"
+  register: need_bootstrap
+  failed_when: false
+  changed_when: false
+  with_items:
+    - python2
+    - pip2
+    - ethtool
+    - python2-netaddr
+  environment: {}
+  tags: facts
+
+- name: Bootstrap | Install python 2.x and pip
+  raw:
+    pacman -Sy && \
+    pacman -S --noconfirm python2 python2-pip ethtool python2-netaddr
```

Comment:
What is the reason to install `ethtool` and `python2-netaddr`?

Reply:
It was present on original repository. Removed the uneeded packages.

Reply:
*unneeded (typo)

Reply:
Ansible uses the python-netaddr package and in the vagrant box archlinux/archlinux the python-netaddr package is not installed, so is need keep the package python-netaddr.

Reply:
> Ansible uses the python-netaddr package

Where? Which module requires this package to be installed on the target machine?

Reply:
Yes you are right you really don't need these packages.
https://docs.ansible.com/ansible/2.7/user_guide/playbooks_filters_ipaddr.html